src/revaultd/mod.rs

```diff
@@ -178,7 +178,14 @@ pub struct ListOnchainTransactionsResponse {
 // RevaultD can start only if a config path is given.
 pub async fn start_daemon(config_path: &Path) -> Result<(), RevaultDError> {
     debug!("starting revaultd daemon");
-    let child = Command::new("revaultd")
+    let mut command = std::path::PathBuf::new();
+    if let Ok(revaultd_path) = std::env::var("REVAULTD_PATH") {
```

Comment:
We try to keep every ENV VAR parsing in main.rs
https://github.com/edouardparis/revault-gui/blob/fix-ci-workflow/src/main.rs#L14
Rename var for REVAULTD_BIN_PATH and store result in app::Config
https://github.com/edouardparis/revault-gui/blob/fix-ci-workflow/src/ui/app.rs#L122
then pass it here: https://github.com/edouardparis/revault-gui/blob/fix-ci-workflow/src/ui/state/charging.rs#L66

Reply:
Fixed, thanks :)